src/doc/users-guide/install.md

```diff
@@ -99,42 +99,54 @@ the `--help` argument to the script.
 
 The OpenCilk C (or C++) compiler can be invoked via `bin/clang` (or
 `bin/clang++`) from within the installation directory.
+Alternatively, you can configure your system so that `clang` and `clang++` point to the OpenCilk C and C++ compilers, 
+as shown in the example below.
 
-{% alert "info" %}
-***Example:***
+### Example
+
+{% alert "primary" %}
 
-The following shows the process of installing OpenCilk into a version-specific
-subdirectory within `/opt/opencilk/`.
+The following example shows the process on Ubuntu 20.04 to install OpenCilk into `/opt/opencilk/`
+without adding a version-specific subdirectory.
+The installation and setup process is analogous for macOS and other Linux systems.
+
+- Download the precompiled [OpenCilk shell
+archive](/doc/users-guide/install/#installing-using-a-shell-archive) for your
+system.  
+- Make directory `/opt/opencilk` if it does not already exist, and execute the shell script to install OpenCilk into the directory.
 
 ```shell-session
-$ sh OpenCilk-2.0.0-x86_64-Linux-Ubuntu-20.04.sh --prefix=/opt/opencilk
+$ mkdir -p /opt/opencilk
+$ sh OpenCilk-2.0.0-x86_64-Linux-Ubuntu-20.04.sh --prefix=/opt/opencilk --exclude-subdir
 OpenCilk Installer Version: 2.0.0, Copyright (c) OpenCilk
 This is a self-extracting archive.
-The archive will be extracted to: /opt/opencilk
-
-If you want to stop extracting, please press <ctrl-C>.
-
-[...license text omitted...]
+The archive will be extracted to: /opt/opencilk 
+Using target directory: /opt/opencilk/
+Extracting, please wait... 
+Unpacking finished successfully
+```
 
-Do you accept the license? [yn]:
-y
+- Set `clang` and `clang++` to point to the OpenCilk C and C++
+compilers.  This can be achieved in numerous ways, such as by setting your
+`PATH` environment variable to look in `/opt/opencilk/bin/`, or by installing
+system-wide symbolic links as:
 
-By default the OpenCilk will be installed in:
-  "/opt/opencilk/OpenCilk-2.0.0-x86_64-Linux-Ubuntu-20.04"
-Do you want to include the subdirectory OpenCilk-2.0.0-x86_64-Linux-Ubuntu-20.04?
-Saying no will install in: "/opt/opencilk" [Yn]:
-y
+```shell-session
+$ sudo ln -s /opt/opencilk/bin/clang /usr/local/bin/clang
```

Comment:
This instruction seems like it could damage a system that already has a `clang` binary, or link to one, at `/usr/local/bin/clang`.  (Likewise with the next instruction for `clang++`.)  Do we have a sense as to how common such links or binaries are on Linux systems?  Do any common Linux distributions put `clang` in `/usr/local/bin/clang`?

Reply:
On Ubuntu, I believe that binaries installed through the package manager all end up in `/usr/bin/`, and `/usr/local/bin/` seems to be reserved for things that take a "custom" installation route.  For example, the LLVM `clang` binary on my system is `/usr/bin/clang`, and my `/usr/local/bin/` folder includes binaries for Matlab, Python's `pip`, Google's `bazel`, etc (which were not installed using `apt`).

I believe that this is a common enough convention but of course I cannot speak to the prevalence of any deviations.

I can see four main approaches for a "default setup example" that could be assumed by all our pages:

1. Use the absolute path (`/opt/opencilk/bin/clang`) everywhere
    - *Pros:* unambiguous, safe
    - *Cons:* a bit unwieldy, may create the feeling of "not really an installed program"
2. Symlinks: `/usr/local/bin/clang`, `/usr/local/bin/clang++`
    - *Pros:* available to all users, `PATH` should include `/usr/local/bin/` by default
    - *Cons:* overrides "vanilla" `clang`, may be destructive on some systems, may silently have no effect on some systems if `PATH` gives precedence to `/usr/bin/` over `/usr/local/bin/`, requires `sudo`
3. Symlinks: `/usr/local/bin/opencilk`, `/usr/local/bin/opencilk++` (or some other non-aliased name)
    - *Pros:* same as (1) and (2), branding
    - *Cons:* may feel less "safe" if the connection to `clang` is obfuscated, requires `sudo`, needs a good name
4. Edited `PATH`, with or without symlinks as in (2) or (3)
    - *Pros:* most flexible, does not require `sudo`
    - *Cons:* most complicated, if someone is comfortable editing their `PATH` they probably already don't need these instructions

Reply:
When I tested the instructions, I just followed along. Then I dug around because `ln` wasn't working as I expected, and I stumbled into the fear... "Did I just damage my system with `ln`?"

So if we stick with symlinks (which makes sense to me), I think it would be good to clue people into this concern.

Reply:
The more I think about it, the less comfortable I am with option 2.  Although I value convenience and user experience highly, I'm very concerned with the risk of damaging someone's system by potentially overwriting entries in `/usr/local/bin`.   Depending on the system, there isn't necessarily a simple way to undo that change.  If the system did put `clang` and `clang++` in `/usr/local/bin` for some reason and the user overwrites those entries by following our instructions, it can become very hard to diagnose the problems that arise and fix them.  This issue is particularly true of compilers.  I personally think the risk is too high.

I think the right way to handle installing things into `/usr/` or its subdirectories is to use the system's package manager and other infrastructure, such as `update-alternatives` on Ubuntu.  However, that's another nontrivial piece of system infrastructure we would have to take on.  For example, after a lot of search, I _think_ I found the build scripts for creating official Debian packages for LLVM here: https://salsa.debian.org/pkg-llvm-team/llvm-toolchain/-/tree/14/debian.  At first glance, it does not seem simple to understand these scripts and adapt them to package OpenCilk similarly to ordinary Clang/LLVM.  If someone wants to take that on, however, let me know.

Right now I'm leaning in favor of option 4.  I agree that option 1 is unwieldy, and I think option 3 has the additional con of potentially confusing users about how to use OpenCilk.  In addition, there are also lots of situations where users don't have `sudo` access.  Option 4 is indeed more complicated, but I'm not convinced that appending a line to a `.bashrc` file is that much more complicated than mucking around inside `/usr/`.  In addition, there's a simple way to undo that change if something does go wrong from appending that line.

Reply:
I am 100% on board with where this discussion is going. Since I want to publish the updated example today, I am going to remove all "dangerous" details about configuring `clang` and `clang++`, leaving instead a reference that we'll say more soon on that topic. That will give us time to work through the details on our agreed approach (which seems to be `PATH`).